### DISCUSSION TITLE
OE4T Appreciation Post

### DISCUSSION THREAD

**Discussion by conradmenchine** [OE4T/meta-tegra#1629] [General]
This is very off-topic, but I'd like to bring up how incredible the OE4T project is.

It's become increasingly clear that the amount of documentation and resources available on this Github project (between the wiki, issues, and discussion posts) far surpasses the level of first-party support provided by Nvidia themselves. In addition to that, I had been repeatedly stumbling across this project for the past few years while working with Xilinx FPGAs, because the yocto support here is _also_ some of the best resources on the internet.

So, thank you @madisongh and to the rest of the contributors here. You are doing the embedded community a wonderful service.

**Comment by ichergui** [comment]
Thanks @conradmenchine for your kind message.
Hopefully, we will continue doing this for the upcoming years

**Reply by madisongh** [reply]
Yes, thanks, @conradmenchine!  It's always nice to hear a thank-you once in a while. :smiley: 

And thanks to everyone who has helped out with patches, wiki updates, and answering questions here and in our Gitter/Matrix channel! You all contribute to the pool of useful knowledge that we're building here.
